RelatedPerson/RelatedPerson-AnitaLambertsDeceasedPatient-Example.json

```diff
@@ -0,0 +1,21 @@
+{
+  "resourceType": "RelatedPerson",
+  "id":"RelatedPerson-AnitaLambertsDeceasedPatient-Example",
```

Comment:
May potentially need an actual Patient resource to collect the details of the related person (even though this isn't collected on the scenario), can discuss